cmd/copy.go

```diff
@@ -427,6 +428,16 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 		cooked.listOfFilesChannel = listChan
 	}
 
+	if raw.includeBefore != "" {
+		// must set chooseEarliest = true, so that if there's an ambiguous local date, the earliest will be returned
+		// (since that's safest for includeBefore.  Better to choose the earlier time and do more work, than the later one and fail to pick up a changed file
+		parsedIncludeBefore, err := includeBeforeDateFilter{}.ParseISO8601(raw.includeBefore, true)
```

Comment:
the scenario is the opposite for include before, chooseEarliest should be false, which is the safer option: we'd do more work rather than failing to pick up a file.

Reply:
Sorry, copy pasta.